src/Middleware/src/Headstart.Common/Assets/english-translations.json

```diff
@@ -7,9 +7,12 @@
     "BUYER_LOCATIONS": "Buyer Groups",
     "BUYERS": "Buyers",
     "SELLER": "Seller",
+    "SELLER_ADMIN": "Seller Admin",
+    "SELLER_LOCATION": "Seller Address",
+    "SELLER_LOCATIONS": "Seller Addresses",
     "SELLERS": "Sellers",
     "SUPPLIER": "Supplier",
-    "SUPPLIER_LOWERCASE": "supplier",
+    "SUPPLIER_LOWERCASE": "Supplier",
```

Comment:
we should change the key name if we want it to not be lowercase. Or are we saying this is "Supplier" instead of "SUPPLIER"?

Reply:
I'm not sure what the point of that one is. I'll remove it